specs/span-batches.md

```diff
@@ -153,6 +154,20 @@ decoding. For example, lets say bad batcher wrote span batch which `block_count
 the explicit limit, not trying to consume data until EOF is reached. We can also safely preallocate memory for decoding
 because we know the upper limit of memory usage.
 
+## Span batch Hard Fork Rule
+
+Span batch hard fork is activated based on timestamp.
+
+Activation Rule: `x != null && x >= upgradeTime && y != null && y >= upgradeTime`
```

Comment:
I'd be tempted to "inline" `x` and `y` here for readability. I'm also not sure how you'd get `null` values for either of them. So I think we could just have:
```suggestion
Activation Rule: `span_start.timestamp >= upgradeTime && inclusion_block.timestamp >= upgradeTime`
```

Then keep the explanations of `span_start.timestamp` and `inclusion_block.timestamp` below.

Reply:
I followed the style of [superchain-upgrades.md#l2-block-timestamp-based-activation](https://github.com/ethereum-optimism/optimism/blob/develop/specs/superchain-upgrades.md#l2-block-timestamp-based-activation) specs. I agree that your suggestion makes readability better, but maybe break the consistency.

Reply:
Fair enough - worth being consistent even if I'm not a fan. :)